### ISSUE TITLE
Brave cannot start

### ISSUE THREAD

**Issue by sangorys** [brave/brave-browser#7914]
## Description 
brave-browser cannot start. Error during launching it

## Steps to Reproduce
   1. run brave in a terminal : brave-browser

## Actual result:
/usr/bin/brave-browser : ligne 48 : 31491 Trappe pour point d'arrêt et de trace              (core dumped) "$HERE/brave" "$@"

then it stops

## Expected result:
brave application is running

## Reproduces how often: 
Always

## Brave version (brave://version info)
brave-browser/eoan,stable,now 1.2.43 amd64

## Version/Channel Information:

- Can you reproduce this issue with the current release? yes
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the dev channel? yes
- Can you reproduce this issue with the nightly channel? yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:
System:
  Host: sangorys-ThinkPad-W530 Kernel: 5.3.0-26-generic x86_64 bits: 64 
  compiler: gcc v: 9.2.1 Desktop: Gnome 3.34.1 
  Distro: Ubuntu 19.10 (Eoan Ermine)

**Comment by kjozwiak** [comment]
@sangorys which source did you use to download Brave?

**Comment by sangorys** [comment]
> @sangorys which source did you use to download Brave?

I installed it with that (and I install all variant dev/nightly/... with the same issue) : 

sudo apt install apt-transport-https curl

curl -s https://brave-browser-apt-release.s3.brave.com/brave-core.asc | sudo apt-key --keyring /etc/apt/trusted.gpg.d/brave-browser-release.gpg add -

echo "deb [arch=amd64] https://brave-browser-apt-release.s3.brave.com/ stable main" | sudo tee /etc/apt/sources.list.d/brave-browser-release.list

sudo apt update

sudo apt install brave-browser

**Comment by akhilsp32** [comment]
I'm having the same issue on Ubuntu 18.04

**Comment by rebron** [comment]
@akhilsp32 can you give us a dump from here:
/.config/BraveSoftware/Brave-Browser/Crash Reports

and can you tell us what version you're running?

**Comment by akhilsp32** [comment]
@rebron `.config/BraveSoftware/Brave-Browser/Crash Reports` is an empty directory for me.

I'm getting this output in my terminal:
```
[21449:21449:0316/175658.587404:ERROR:vaapi_wrapper.cc(437)] The system version 1.1 should be greater than or equal to 1.5
[21449:21449:0316/175658.657890:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
[21490:1:0316/175659.452758:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[21490:1:0316/175659.453191:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[21490:1:0316/175659.912930:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[21490:1:0316/175659.913370:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
/usr/bin/brave-browser: line 48: 21424 Bus error               (core dumped) "$HERE/brave" "$@"
```

Version of brave-browser is `80.1.4.96`

I'm running on Ubuntu 18.04.4 LTS

**Comment by fmarier** [comment]
@akhilsp32 You're using the same version of Ubuntu as me and so I am wondering whether there is something broken in your existing browser profile.

@akhilsp32 and @sangorys, could you please try the following:

1. Install the latest version of the `brave-browser` package (i.e. not beta, dev, or nightly, but release) using our [official instructions](https://brave-browser.readthedocs.io/en/latest/installing-brave.html#debian-9-ubuntu-14-04-and-mint-17).
2. Run it with an empty profile like this:
```
mkdir $HOME/BraveTemp
brave-browser-stable --user-data-dir=$HOME/BraveTemp
```

Does it crash on startup?

**Comment by akhilsp32** [comment]
Thanks @fmarier. Now it works!

However my bookmarks are gone. Is there any way I can restore them too?

**Comment by bbugh** [comment]
The latest release version (from the downloads page) before v1.5.112 was crashing on MacOS 10.14.6, but v1.5.112 is no longer crashing on startup. Thanks!

**Comment by fmarier** [comment]
> However my bookmarks are gone. Is there any way I can restore them too?

Great, now I suggest you do the following:

1. Close Brave.
2. Backup your existing profile directory:
```
mv ~/.config/BraveSoftware/Brave-Browser ~/.config/BraveSoftware/Brave-Browser.backup
```
3. Start Brave to create a new profile directory.
4. Close Brave.
5. Restore your old bookmarks:
```
cp ~/.config/BraveSoftware/Brave-Browser.backup/Default/Bookmarks ~/.config/BraveSoftware/Brave-Browser/Default/Bookmarks
```

After that, you should be able to start Brave without problems and see your bookmarks. Let me know how that goes.

**Comment by bbugh** [comment]
This is happening to me again today suddenly. I can't open Brave to see which version it is. I tried the suggested profile removal as mentioned above (on Mac it's `/Users/USER/Library/Application Support/BraveSoftware`) but it didn't work.

**Comment by Nuc1eoN** [comment]
I've had a similar issue in #8916

Where can we find the core dump?

**Comment by bsclifton** [comment]
A potential fix went out today with version 1.7

Can folks please try to download the latest Brave and give it a go? If you still experience a crash and may have had sync enabled, you can try launching with the command line args `--disable-sync`

Please let us know!
@Nuc1eoN @bbugh @sangorys @akhilsp32

**Comment by bsclifton** [comment]
Closing as root issue reported seems to have been resolved

There is an ongoing investigation into hanging/freezing behavior which I would recommend folks subscribing to / leaving a comment if you're experiencing a problem
https://github.com/brave/brave-browser/issues/9481
